extensions/amp-story/1.0/amp-story.css

```diff
@@ -455,6 +455,23 @@ amp-story-grid-layer .i-amphtml-embedded-component::after {
   height: var(--i-amphtml-story-layer-height, 100%);
   font-size: calc(var(--i-amphtml-story-layer-height, 100vh) / 10);
   margin-left: calc((var(--story-page-vw, 1%) * 100 - var(--i-amphtml-story-layer-width, 100%)) * 0.5); /* Set margin to 1/2 of page-width - layer-width to center layer */
+
+  --i-amphtml-aspect-ratio-calculated: calc(var(--aspect-ratio)) !important; /* Avoid fractions, which don't work with compilers that remove parenthesis */
+  --i-amphtml-story-page-width: calc(100 * var(--story-page-vw, 1vw)) !important;
+  --i-amphtml-story-page-height: calc(100 * var(--story-page-vh, 1vh)) !important;
```

Comment:
Plz update these variable names, it's too easy to get into collisions at some point with the `one-panel-page-height` etc

Reply:
If they collide, they would have exactly the same value (as it's literally getting the width and height of the page regardless of UI type), and I could just remove the usage from here. I'm open to moving them up to an ancestor of the grid-layers also, but they aren't currently needed anywhere else afaik. Still think it's worth it to rename it to sth like `--i-amphtml-story-ancestor-page-height` or similar?